site/public/mjs/sql-toolbox.js

```diff
@@ -73,3 +73,9 @@ async function runSqlQuery() {
         console.error(exc);
     }
 }
+
+export function init() {
+    document.getElementById('run-sql-btn').addEventListener('click', () => runSqlQuery());
```

Comment:
JS modules are not able to be used directly within HTML like scripts are, so we have to manually build our event listener bindings. Not a huge deal, but something to be aware of in the transition.